keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/plugins/ArtifactSupplier.kt

```diff
@@ -34,11 +35,11 @@ interface ArtifactSupplier<A : DeliveryArtifact, V : VersioningStrategy> : Spinn
    * persisted and evaluated for promotion into deployment environments.
    *
    * The default implementation of [publishArtifact] simply publishes the event via the [EventPublisher],
-   * and should *not* be overridden by implementors.
+   * and should typically *not* be overridden by implementors.
    */
   @JvmDefault
-  fun publishArtifact(artifactPublishedEvent: ArtifactPublishedEvent) =
-    eventPublisher.publishEvent(artifactPublishedEvent)
+  fun publishArtifact(artifact: PublishedArtifact) =
```

Comment:
why was the event name changed?

Reply:
I didn't change the event name, just the type that's passed into the method. Instead of passing an event with multiple artifacts, I'm now passing a single artifact, which is what this method is intended to process.